src/components/BankAccount/index.js

```diff
@@ -1,11 +1,24 @@
+import useStore from '../../hooks/useStore';
+
 import {Section} from './styled';
 
-export default function BankAccount({accountName, bankName, accountValue}) {
+export default function BankAccount() {
+	const bankAccounts = useStore(state => state.db);
+	const deleteBankaccount = useStore(state => state.deleteBankaccount);
 	return (
-		<Section>
-			<h2>{accountName}</h2>
-			<h3>{bankName}</h3>
-			<p>{accountValue} €</p>
-		</Section>
+		<>
+			{bankAccounts.map(bankAccount => {
+				return (
+					<Section key={bankAccount.id}>
+						<h2>{bankAccount.accountName}</h2>
+						<h3>{bankAccount.bankName}</h3>
+						<p>{bankAccount.accountValue} €</p>
+						<button type="button" onClick={() => deleteBankaccount(bankAccount.id)}>
+							Delete
+						</button>
+					</Section>
```

Comment:
Just a minor comment: 
You could think about your individual accounts as articles `<article></article>.` But this is really just a suggestion and perhaps also a question of _"semantic taste"_. 🍩

Reply:
Thanks for your feedback! We will do a bit of research on the article and section tag and then decide what would be best to use in our case!